app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,6 +26,7 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        dd($signup->events);
```

Comment:
Yeah, I'm a little confused by the relationship between events and signups. The way we're trying to set a single `event_id` column in the Signup factory makes sense for a "belongs to" relationship (where each signup belongs to a single event).

For a "has many" relationship, I think you'd want to do the chained `->each()` calls on the returned factory to make the events like you suggested on Slack.